docs/concepts-messaging.md

```diff
@@ -244,6 +244,13 @@ The default retry letter topic uses this format:
 <topicname>-<subscriptionname>-RETRY
 ```
 
+Note: In versions `2.6.x` and `2.7.x`, the default retry letter topic uses this format is `<subscriptionname>-RETRY`.
+If you are upgrading from version `2.6.x~2.7.x` to version `2.8.x` or greater, then you need to manually delete
+the historical retry letter topics and retry letter partitioned topics. Otherwise, Pulsar will continue to use the
+original topics, which were formatted with `<subscriptionname>-RETRY`. Rule `<subscriptionname>-RETRY` should be avoided
+to use, because if multiple topics under the same namespace have the same subscription, the retry message topic names
+for multiple topics will be the same, resulting in mutual consumption.
```

Comment:
```suggestion
:::note

- For Pulsar 2.6.x and 2.7.x, the default retry letter topic uses the format of `<subscriptionname>-RETRY`. If you upgrade from 2.6.x~2.7.x to 2.8.x or later, you need to delete historical retry letter topics and retry letter partitioned topics. Otherwise, Pulsar continues to use original topics, which are formatted with `<subscriptionname>-RETRY`. 

- It is not recommended to use `<subscriptionname>-RETRY` because if multiple topics under the same namespace have the same subscription, then retry message topic names for multiple topics might be the same, which will result in mutual consumptions.

:::
```

Reply:
Fixed

Reply:
<img width="991" alt="截屏2023-07-28 11 15 29" src="https://github.com/apache/pulsar-site/assets/25195800/4a083e07-1ef2-4b58-9552-147cb4aa4790">